File: Python/soma_vetor.py
#  ***********************************************************
#  * Descrição:
#  * Faça um programa que leia N números reais e armazene-os
#  * em um vetor. Em seguida, imprimir todos os elementos do
#  * vetor, e mostrar na tela a soma e a média dos elementos.
#  ***********************************************************

cases = int(input("Quantos números serão digitados? "))

vetor = [0 for x in range(cases)]
soma = 0

for i in range(cases):
    vetor[i] = float(input("Digite um número: "))
    soma += vetor[i]

media = soma / cases

print("Valores ", end="")
for i in range(cases):
    print(f"{vetor[i]:.1f}", end=" ")

print()
print(f"Soma = {soma:.2f}")
print(f"Média = {media:.2f}")

File: Java/somaVetor.java
/*
 ***********************************************************
 * Descrição:
 * Faça um programa que leia N números reais e armazene-os
 * em um vetor. Em seguida, imprimir todos os elementos do
 * vetor, e mostrar na tela a soma e a média dos elementos.
 ***********************************************************
 */

import java.util.Scanner;

public class somaVetor {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int cases;
        double media, soma;

        System.out.print("Quantos números você vai digitar? ");
        cases = sc.nextInt();

        double[] vetNum = new double[cases];

        soma = 0;

        for (int i = 0; i < cases; i++) {
            System.out.print("Digite um número: ");
            vetNum[i] = sc.nextDouble();
            soma = soma + vetNum[i];
        }

        sc.close();

        media = soma / cases;

        System.out.print("Valores = ");

        for (int i = 0; i < cases; i++) {
            System.out.print(vetNum[i] + " ");
        }

        System.out.println();
        System.out.println("Soma = " + String.format("%.2f", soma));
        System.out.println("Media = " + String.format("%.2f", media));
    }
}
